projects/dev-schematics/src/ng-add/index.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  chain,
+  noop,
+  Rule,
+  schematic,
+  SchematicContext,
+  SchematicsException,
+  Tree,
+} from '@angular-devkit/schematics';
+import { getSpartacusVersion } from '../shared/utils/package-utils';
+
+export default function (options: any): Rule {
```

Comment:
Can we specify a type for `options`? Would `DevSpartacusOptions` work? Or maybe `Schema`?